governance.md

```diff
@@ -1,12 +1,12 @@
-The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the technical oversight committee. The steering committee manages the governance structure of the project, revisiting and refining it as needs evolve.
+The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the Technical Oversight Committee (TOC). The TOC manages the governance structure of the project, revisiting and refining it as needs evolve.
 
 The broader community is organized around a set of special interest groups (SIGs) focused on particular areas of interest.
 
 ## Roles
 
 To make clearer to community members how they might progressively get more involved and take more ownership in the project, we define the following appointed roles with their respective duties and qualifications.
 
-Roles are appointed by the Technical Oversight Committee (TOC). Roles can be revoked for reasons such as falling out of qualification, inactivity, or for violations of community guidelines.
+Roles are appointed by the TOC. Roles can be revoked for reasons such as falling out of qualification, inactivity, or violations of community guidelines.
```

Comment:
Should still confirm that TOC means "Technical Oversight Committee"

Reply:
@jasonmcintosh looks like we define TOC in the first line of this doc. Seems reasonable to refer to it as TOC from that point on.